packages/fscomponents/src/components/serializable/vNext/SerializableImage.tsx

```diff
@@ -14,16 +14,27 @@ export interface SerializableImageProps
     | 'progressiveRenderingEnabled'
     | 'resizeMethod'
     | 'resizeMode'
-    | 'style'
     | 'testID'
   > {
   uri?: string;
-  height?: number;
-  width?: number;
-  style?: ImageStyle;
+  style?: ViewStyle;
+  imageStyle?: ImageStyle;
+  children?: React.ReactNode;
 }
 
-export const SerializableImage: FC<SerializableImageProps> = ({ uri, height, width, ...props }) => {
-  const source = useMemo(() => ({ uri, height, width }), [uri, height, width]);
-  return <Image {...props} source={source} />;
-};
+export const SerializableImage: FC<SerializableImageProps> =
+  ({ children, style, uri, ...props }) => {
+    const source = useMemo(() => ({ uri }), [uri]);
+
+    return (
+      <View style={style}>
+        <ImageBackground
+          {...props}
+          source={source}
+          style={[{ flex: 1 }, style]}
+        >
```

Comment:
It looks like we are using the `style` prop on two different components.
How this this interact when doing margins or padding?
Do they get doubled unexpectedly?